spec/ddtrace/contrib/ethon/easy_patch_spec.rb

```diff
@@ -0,0 +1,194 @@
+require 'ethon'
+require 'ddtrace/contrib/ethon/easy_patch'
+require 'ddtrace/contrib/ethon/shared_examples'
+require 'ddtrace/contrib/analytics_examples'
+
+RSpec.describe Datadog::Contrib::Ethon::EasyPatch do
+  let(:tracer) { get_test_tracer }
+  let(:configuration_options) { { tracer: tracer } }
+
+  before do
+    Datadog::Contrib::Ethon::Patcher.patch
```

Comment:
I don't think this should be necessary: `c.use` will call `patch` automatically for the integration, and I think we want to verify that users don't need to manually patch.